Makefile

```diff
@@ -1,37 +1,26 @@
-.PHONY: all clean install build
-all: build doc
+.PHONY: build release install uninstall clean test doc reindent
 
-NAME=vhd-format
-J=4
+build:
+	jbuilder build @install --dev
 
-export OCAMLRUNPARAM=b
+release:
+	jbuilder build @install
 
-setup.bin: setup.ml
-	@ocamlopt.opt -o $@ $< || ocamlopt -o $@ $< || ocamlc -o $@ $<
-	@rm -f setup.cmx setup.cmi setup.o setup.cmo
-
-setup.data: setup.bin
-	@./setup.bin -configure --enable-tests
-
-build: setup.data setup.bin
-	@./setup.bin -build -j $(J)
-
-doc: setup.data setup.bin
-	@./setup.bin -doc -j $(J)
-
-install: setup.bin
-	@./setup.bin -install
+install:
+	jbuilder install
 
 uninstall:
-	@ocamlfind remove $(NAME) || true
+	jbuilder uninstall
 
-test: setup.bin build
-	@./setup.bin -test
+clean:
+	jbuilder clean
 
-reinstall: setup.bin
-	@ocamlfind remove $(NAME) || true
-	@./setup.bin -reinstall
+test:
+	jbuilder runtest
 
-clean:
-	@ocamlbuild -clean
-	@rm -f setup.data setup.log setup.bin
+# requires odoc
+doc:
+	jbuilder build @doc
+
+reindent:
+	git ls-files '*.ml' '*.mli' | xargs ocp-indent --syntax cstruct --syntax lwt -i
```

Comment:
I think the —syntax thing is for camlp4. Do we need it? I think we can also get rid of it in xapi’s packages. But I did not double check

Reply:
I don't think we need any of the `--syntax` params. It indented the code using `[%%cstruct` in a sensible way without them (I think the same way as with these params). I'll remove them for now.